cmd/easeprobe/main.go

```diff
@@ -31,17 +32,24 @@ import (
 	log "github.com/sirupsen/logrus"
 )
 
+func getEnvOrDefault(key, defaultValue string) string {
+	if value, ok := os.LookupEnv(key); ok {
+		return value
+	}
+	return defaultValue
+}
+
+func isExternalLink(value string) bool {
+	_, err := url.Parse(value)
+	return err == nil
+}
+
 func main() {
 
-	dryNotify := flag.Bool("d", false, "dry notification mode")
-	yamlFile := flag.String("f", "config.yaml", "configuration file")
+	dryNotify := flag.Bool("d", os.Getenv("DRY") == "true", "dry notification mode")
+	yamlFile := flag.String("f", getEnvOrDefault("CONFIG", "config.yaml"), "configuration file")
```

Comment:
How about using the `PROBE_` prefix for the environment variable? this could avoid variable name conflicts.